server/chat-commands/moderation.ts

```diff
@@ -865,8 +865,9 @@ export const commands: ChatCommands = {
 		}
 		this.globalModlog(`UNLOCK${range ? 'RANGE' : 'IP'}`, target, ` by ${user.name}`);
 
-		const broadcastRoom = Rooms.get('staff') || room;
-		broadcastRoom.addByUser(user, `${user.name} unlocked the ${range ? "IP range" : "IP"}: ${target}`);
+		if (!room || room?.roomid !== 'staff') {
+			Rooms.get('staff')?.addByUser(user, `${user.name} unlocked the ${range ? "IP range" : "IP"}: ${target}`);
+		}
```

Comment:
This didn't need to be changed. The reason our other code has a staff check is because they already notify the current room. This command specifically doesn't notify the current room, so it should always notify Staff instead.

```suggestion
		const logRoom = Rooms.get('staff') || room;
		logRoom?.addByUser(user, `${user.name} unlocked the ${range ? "IP range" : "IP"}: ${target}`);
		if (!logRoom || logRoom !== room) {
			this.privateModAction(`(${user.name} unlocked the ${range ? "IP range" : "IP"}: ${target})`);
		}
```